libs/community/langchain_community/utilities/brave_search.py

```diff
@@ -48,8 +50,8 @@ def download_documents(self, query: str) -> List[Document]:
         results = self._search_request(query)
         return [
             Document(
-                page_content=item.get("description"),  # type: ignore[arg-type]
-                metadata={"title": item.get("title"), "link": item.get("url")},
+                page_content=item.get("extra_snippets"),  # type: ignore[arg-type]
```

Comment:
is `extra_snippets` a list of strings? If so, is your proposal to collapse them into a single string for page_content?